modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -46,6 +47,34 @@ sda.enkf.multisite <- function(settings,
   ###-------------------------------------------------------------------###
   ### read settings                                                     ###
   ###-------------------------------------------------------------------###
+  
```

Comment:
Instead of this whole piece I think you could make the `SDA` name for the folder to be an argument instead of being hardcoded. It would do what you wanted I guess.